llvm/utils/UpdateTestChecks/common.py

```diff
@@ -388,7 +388,11 @@ def itertests(
 
 
 def should_add_line_to_output(
-    input_line, prefix_set, skip_global_checks=False, comment_marker=";"
+    input_line,
+    prefix_set,
+    skip_global_checks=False,
+    skip_same_checks=False,
+    comment_marker=";",
```

Comment:
```suggestion
    input_line,
    prefix_set,
    *,
    skip_global_checks=False,
    skip_same_checks=False,
    comment_marker=";",
```

Since you added a new parameter it seems like no callers actually pass `comment_marker` here?

I find function calls with various boolean flags quite hard to parse, maybe make these parameters named arguments?

Reply:
Yeah, the call in `update_test_checks.py` is the only one passing anything other than `input_line` and `prefix_set`. The comment marker is always `";"` currently.